### ISSUE TITLE
xtouch script issues

### ISSUE THREAD

**Issue by Badmanpeen** [bramdebouvere/fltouch#19]
hey, I've followed all of the steps on the github but there is no folder "fltouch" in my hardware holder.
I just created and named the folder, then populated it wit the filed from the zip folder.
'fltouch xtouch (user) showed up in controller type in fl settings, but when I selected it- the xtouch immediately disconnects showing blank screens..
I'd love this to work, as the color organization is a huge part of my workflow.
any help is appreciated.
(btw listened to your work on spotify, Bram, Dope!)

**Comment by bramdebouvere** [comment]
I've already replied to this on the IL forum, but just in case you haven't seen it yet, CBaum posted the following:

> X-Touch script on port 103, not Mackie control on port 103.

That's probably why you are getting the issue. Please try to make sure the MIDI settings are correct (see this GitHub readme for more info about that).

Please let me know if that works for you.

**Comment by Badmanpeen** [comment]
thanks for the reply.
unfortunately that was definitely part of the combinations I tried, however to no avail.
i apologize for including the screenshot I took while flailing around trying to get it to work.
here are the settings suggested, however i cannot get the xtouch to connect unless I use the native MCU controller script FL20 comes with. that includes this updated MCU script. (https://forum.image-line.com/viewtopic.php?t=228188)
